node/main-extract-async.ts

```diff
@@ -310,3 +318,54 @@ export function extractAnyMissingThumbs(
     }
   });
 }
+
+/**
+ * Scan the output directory and delete any file not in `hashesPresent`
+ * @param hashesPresent
+ * @param outputDir
+ */
+export function removeThumbnailsNotInHub(hashesPresent: Map<string, number>, outputDir: string): void {
```

Comment:
@cal2195 -- thank you for implementing `chokidar` and `async` 🙇 

This combo makes me feel like a 10x coder 🤩 -- unsure if this is how you would have implemented the feature -- but I add a watcher to look through all the screenshots, and any that don't match an existing hash get sent to a queue runner (`deleteThumbQueue`) to get deleted 😁 

Nice and smooth -- and I think without bugs 😅